src/js/models/messages.js

```diff
@@ -44,4 +44,19 @@ export default class Messages{
 			alert(error);
 		}
 	}
+
+	async getSearchResults(searchContent) {
```

Comment:
We should really separate contacts and messages, these are two different components...

Reply:
Issue created to resolve comment